client/state/email-accounts/schema.js

```diff
@@ -0,0 +1,45 @@
+export const usersSchema = {
+	type: 'array',
+	items: {
+		oneOf: [
+			{
+				type: 'object',
+				required: [
+					'email',
+					'mailbox',
+					'domain',
+					'firstname',
+					'lastname',
+					'fullname',
+					'site_id',
+					'suspended',
+					'product_name',
+					'product_slug',
+				],
+				properties: {
+					mailbox: { type: 'string' },
+					domain: { type: 'string' },
+					email: { type: 'string' },
+					firstname: { type: 'string' },
+					lastname: { type: 'string' },
+					fullname: { type: 'string' },
```

Comment:
Minor but since we're introducing a new endpoint I would suggest using `first_name`, `last_name`, and `full_name` instead. As an aside, do we really need to expose the first and last name? What if the API simply returned a `name` property (which would contain the full name)?

Reply:
Great suggestion as usual. If you think about it though, `full_name` is mostly a duplication, and we should leave that out. However I think `first_name` and `last_name` should remain, because provider APIs often provide them e.g. G Suite.